client/src/models/Report.js

```diff
@@ -400,8 +452,29 @@ export default class Report extends Model {
     }
   }
 
-  static getPrimaryAttendee(attendees, role) {
-    return attendees.find(el => el.role === role && el.primary)
+  static checkAttendingAuthor(reportPeople) {
+    if (!reportPeople.some(rp => rp.author && rp.attendee)) {
+      return "You must provide at least 1 attending author"
+    }
+  }
+
+  static checkAnyAuthor(reportPeople) {
+    if (!reportPeople.some(rp => rp.author)) {
```

Comment:
Even though, currently, `reportPeople` parameter seems to be always defined wherever this function is being called, it would be safer to add null checks with optional chaining.